frappe/cache_manager.py

```diff
@@ -141,17 +141,13 @@ def build_table_count_cache():
 		return
 
 	_cache = frappe.cache()
-	data = frappe.db.multisql({
-		"mariadb": """
-			SELECT 	table_name AS name,
-					table_rows AS count
-			FROM information_schema.tables""",
-		"postgres": """
-			SELECT 	"relname" AS name,
-					"n_tup_ins" AS count
-			FROM "pg_stat_all_tables"
-		"""
-	}, as_dict=1)
+	name = frappe.qb.Field("table_name").as_("name")
+	count = frappe.qb.Field("table_rows").as_("count")
+	information_schema = frappe.qb.Schema("information_schema")
+
+	q = frappe.qb.from_(information_schema.tables).select(name, count).get_sql()
```

Comment:
Instead of having a trailing `get_sql` in every query, we should probably just type cast every query as str inside `frappe.db.sql`, given `frappe.qb` wont always return an object inheriting from a single class at any level. 🤔